zenpacklib.py

```diff
@@ -1625,7 +1625,8 @@ def get_bases(cls, bases=[]):
 
                     # make sure we have the schema defined
                     if target_relname in class_.relationships:
-                        class_.relationships[target_relname].schema = target_schema
+                         if not class_.relationships[target_relname].schema:
+                             class_.relationships[target_relname].schema = target_schema
```

Comment:
zenpacklib.py|1628 col 26 error| indentation is not a multiple of four [E111]                      
zenpacklib.py|1629 col 30 error| indentation is not a multiple of four [E111]